Source/SwiftLintBuiltInRules/Rules/Idiomatic/RedundantTypeAnnotationRule.swift

```diff
@@ -1,9 +1,143 @@
 import Foundation
-import SourceKittenFramework
+import SwiftLintCore
+import SwiftSyntax
 
-struct RedundantTypeAnnotationRule: OptInRule, SubstitutionCorrectableRule {
-    var configuration = SeverityConfiguration<Self>(.warning)
+struct RedundantTypeAnnotationRule: SwiftSyntaxCorrectableRule, OptInRule {
```

Comment:
Please use `@SwiftSyntaxRule` and remove the explicit `makeVisitor` implementation.